Program/stud1.txt

```diff
@@ -0,0 +1,41 @@
+[1] Create table stud(id, name, dept, paidfee) with 8 records. Create stud.csv file of the stud data (Department will be BBA and BCA). 
+   
+   Read csv file using reader object and write data with remaining fee. ( Total fee of bba is 13400 and bca is 15400 Rs)
+
+
+sqlite> .open student

```

Comment:
here you opened student database? Any other way to open or create a database?

Reply:
> here you opened student database? Any other way to open or create a database?

Yes we use " sqlite3 your_database_name.db " to open the databse in another way.